### PR TITLE
Update voc files

### PR DESCRIPTION
**Pull request by anatheka**

### CODE REVIEW

File: service/discovery/azure/azure.go [hunk 1]
```diff
@@ -304,16 +305,8 @@ func (d *azureDiscovery) discoverBackupVaults() error {
 				// Store voc.Backup in backupMap
 				d.backupMap[util.Deref(instance.Properties.DataSourceInfo.DatasourceType)][util.Deref(instance.Properties.DataSourceInfo.ResourceID)] = &voc.Backup{
 					Enabled:         true,
-					Policy:          util.Deref(instance.Properties.PolicyInfo.PolicyID),
 					RetentionPeriod: retentionDuration(util.Deref(retention)),
 					Storage:         voc.ResourceID(util.Deref(instance.ID)),
-					GeoLocation: voc.GeoLocation{
```
Comment (oxisto) [inline service/discovery/azure/azure.go:310]: Any particular reason for deleting this?

File: voc/firewall.go [hunk 2]
```diff
@@ -1,6 +1,6 @@
 // Auto-generated code by owl2java (https://github.com/clouditor/cloud-property-graph)
 
-// Copyright 2023 Fraunhofer AISEC
+// Copyright 2022 Fraunhofer AISEC
```
Comment (oxisto) [inline voc/firewall.go:3]: Moving back in time?

**Review by oxisto — APPROVED** [review]
Just a question about why something was deleted, otherwise approved.

File: service/discovery/azure/azure.go (hunk 1 above)
Comment (anatheka) [reply service/discovery/azure/azure.go:310]: The geolocation can also be found in the corresponding resource object.

**Pull request merged**